vng_api_common/models.py

```diff
@@ -166,9 +84,9 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
     @classmethod
-    def get_client(cls) -> Optional[Client]:
+    def get_client(cls) -> APIClient | Any | None:
```

Comment:
Whenever you set `CLIENT_CLASS` anything other than the default, it will not return the default (`APIClient` at this time of writing)

Reply:
I guess the underlying question is if we want users to allow them to use clients that do not inherit from `APIClient`. If we want to allow users to do this, `Any` would be correct here, otherwise it should not be typed like that.

Reply:
I don't think we'll be using clients that do not inherit from `APIClient`, so let's use `APIClient | None`